build.sbt

```diff
@@ -1,16 +1,31 @@
 // See LICENSE for license details.
 
-import Dependencies._
+name := "barstools"
 
-lazy val commonSettings = Seq(
-  organization := "edu.berkeley.cs",
-  version := "0.1-SNAPSHOT",
-  scalaVersion := "2.11.8",
-  libraryDependencies ++= commonDependencies
+organization := "edu.berkeley.cs"
+
+version := "1.0"
+
+scalaVersion := "2.11.7"
+
+resolvers ++= Seq (
+  Resolver.sonatypeRepo("snapshots"),
+  Resolver.sonatypeRepo("releases")
+)
+
+libraryDependencies ++= Seq("chisel3","chisel-iotesters").map {
+  dep: String => "edu.berkeley.cs" %% dep % sys.props.getOrElse(dep + "Version", defaultVersions(dep))
+}
+
+scalacOptions ++= Seq("-unchecked", "-deprecation", "-feature", "-language:reflectiveCalls")
+
+// Provide a managed dependency on X if -DXVersion="" is supplied on the command line.
+val defaultVersions = Map(
+  "chisel3" -> "3.1-SNAPSHOT",
+  "chisel-iotesters" -> "1.2-SNAPSHOT"
 )
 
-lazy val executionoptions = (project in file("executionoptions"))
-  .settings(commonSettings)
-  .settings(
-    libraryDependencies ++= executionoptionsDependencies
-  )
+libraryDependencies ++= Seq(
+  "org.scalatest" %% "scalatest" % "2.2.5",
+  "org.scalacheck" %% "scalacheck" % "1.12.4"
+)
```

Comment:
While this is probably okay right now, we're going to need to make this a multi-project build in the near future. We would like to factor out some common utils from Chisel and Firrtl here so they can both depend on it (which means there needs to be a subproject for things that depend on chisel3 and a different subproject for things that don't). As long as we don't publish this globally it's fine, but basically we probably need a more specific project name for these things that depend on chisel3.